src/netbsd.rs

```diff
@@ -24,28 +44,33 @@ fn kern_arnd(buf: &mut [MaybeUninit<u8>]) -> libc::ssize_t {
 
 type GetRandomFn = unsafe extern "C" fn(*mut u8, libc::size_t, libc::c_uint) -> libc::ssize_t;
```

Comment:
Should the first argument be `*mut c_void` here? That's what the man pages has: https://man.freebsd.org/cgi/man.cgi?query=getrandom&sektion=2

I'm not sure if it particularly matters

Reply:
> I'm not sure if it particularly matters

In theory it could matter depending on the CFI rules. I changed it.